tests/unit/python/fledge/common/test_configuration_manager.py

```diff
@@ -2111,11 +2479,19 @@ def mock_coro():
         async def async_mock(return_value):
             return return_value
 
-        attrs = {"delete_from_tbl.return_value": mock_coro()}
+                # Changed in version 3.8: patch() now returns an AsyncMock if the target is an async function.
+        if sys.version_info.major == 3 and sys.version_info.minor >= 8:
+            _attr = await mock_coro()
```

Comment:
Class dict does not define await, so the await operator cannot be used on its instances.
Remove asyncio.coroutine from the mock_coro def and make it as async

Reply:
Marking async def x is good and clean. But that marker itself makes it async so kept as is. This will make it behave like a future task hence async await paradigm; else it would have complained as not awaitable